src/HttpUpgradeProtocolAccess.h

```diff
@@ -114,7 +114,14 @@ HttpUpgradeProtocolAccess::forApplicable(const ProtocolView &offer, const Visito
         seenApplicable = true; // may already be true
     }
     if (!seenApplicable && other) // OTHER is applicable if named rules were not
-        (void)visitor(ProtoOther, other);
+        (void)visitor(ProtoOther(), other);
+}
+
+inline const SBuf &
```

Comment:
We can make this auto as well. We do not have a policy about it (yet?). I do not insist on this change. Your call.

```suggestion
inline const auto &
```

Reply:
I thought about it. The declaration needs it, and at that point I'd rather keep a consistent signature in the implementation, even if in theory we could make it auto